sys-block/zram-init/zram-init-11.1.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit prefix readme.gentoo-r1
+
+DESCRIPTION="Scripts to support compressed swap devices or ramdisks with zRAM"
+HOMEPAGE="https://github.com/vaeth/zram-init/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vaeth/${PN}.git"
+else
+	SRC_URI="https://github.com/vaeth/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~riscv ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+RESTRICT="binchecks strip test"
```

Comment:
Not sure I get the need to add any of this? It should be fine without.

Reply:
I'm not sure when to add this. I thought it was a good practice for non-binary packages to add `RESTRICT="binchecks strip"` and the same for packages without test to add `RESTRICT="test"`.